src/bkl/plugins/vs201x.py

```diff
@@ -348,7 +359,7 @@ def gen_for_target(self, target, project):
         filename = project.projectfile.as_native_path_for_output(target)
         paths_info = self.get_project_paths_info(target, project)
 
-        formatter = XmlFormatter(target.project.settings, paths_info)
+        formatter = VS201xXmlFormatter(target.project.settings, paths_info)
```

Comment:
Why? `XmlFormatter` should already be this one (it’s a class variable), using it seems safer - if some future derived class needs further tweaks, they’ll be picked up automatically.